xbmc/addons/Addon.cpp

```diff
@@ -36,6 +36,59 @@
 using XFILE::CDirectory;
 using XFILE::CFile;
 
+namespace
+{
+class FilenameXMLCache
```

Comment:
Class names should start with an upper "C" => https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#85-classes